libr/anal/esil.c

```diff
@@ -1686,7 +1686,7 @@ static int esil_poke_some(RAnalEsil *esil) {
 	ut64 ptr, regs = 0, tmp;
 	char *count, *dst = r_anal_esil_pop (esil);
 	if (dst && r_anal_esil_get_parm_size (esil, dst, &tmp, &regsize)) {
-		ut8 bytes = regsize / 8;
+#define bytes regsize / 8
```

Comment:
also. we dont want this. we know that the max regsize is 64.. at least for now, so you can just hardcode this. otherwise making it a define will not solve the vararray issue